### PR TITLE
Log-in View and User model

### PR DESCRIPTION
**Pull request by anaimhunter**
Here we change the design of the login view design. For the user we create a class with two constructors and also create a method that assigns the values of the credentials.

### CODE REVIEW

**Comment by ivanebernal** [comment]
I see that you have 2 PRs open for the same change. If this is intentional, please explain why in the description, otherwise close one of the PRs and open one for each change you intended to make.

**Comment by anaimhunter** [comment]
> I see that you have 2 PRs open for the same change. If this is intentional, please explain why in the description, otherwise close one of the PRs and open one for each change you intended to make.

Yes they are two because each one is on diferent branch.

**Comment by ivanebernal** [comment]
Okay. From now on, we will create a new branch for each new feature/fix.

**Comment by anaimhunter** [comment]
image

File: .gitignore [hunk 1]
```diff
@@ -202,4 +202,7 @@ fabric.properties
 # Google services
 app/google-services.json
 
+#Images
+app/src/main/res/drawable/teckerslogo.png
```
Comment (ivanebernal) [inline .gitignore]: Please remove these lines, images are not files we should ignore.

**Pull request merged**